src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationApplicationsController.kt

```diff
@@ -29,7 +29,7 @@ const val APPLICATION_CHANGE: String = "$ORCHESTRATION_APPLICATIONS_CONTROLLER_P
 class OrchestrationApplicationsController(
   private val applicationsService: ApplicationsService,
 ) {
-  @PreAuthorize("hasRole('VISIT_SCHEDULER')")
+  @PreAuthorize("hasAnyRole('VISIT_SCHEDULER', 'VSIP_ORCHESTRATION_SERVICE')")
```

Comment:
Added VSIP_ORCHESTRATION_SERVICE role which will later become the only supported role. Will remove  VISIT_SCHEDULER role as part of a different PR once the front end clients have been updated and tested.